notifications_utils/recipients.py

```diff
@@ -553,13 +553,13 @@ def validate_uk_phone_number(number):
     number = normalise_phone_number(number).lstrip(uk_prefix).lstrip("0")
 
     if not number.startswith("7"):
-        raise InvalidPhoneError("Not a UK mobile number")
+        raise InvalidPhoneError("This is not a UK mobile number")
 
     if len(number) > 10:
-        raise InvalidPhoneError("Too many digits")
+        raise InvalidPhoneError("Mobile phone number cannot be longer than 10 digits")
```

Comment:
I think this may be confusing.

My phone number is 11 digits long. It is of the format 07123456789 and that is 11 digits.

Equally, I think it would be fair to say my phone number is 12 digits long if I wrote it as 447123456789.

Whilst these validation messages appear to be doing the right thing ie being explicit about what exactly is wrong, I'm cautious that because we do a lot of normalisation (removing 0s, symbols, 44 etc) before validating the phone number, the thing we are validating is not the phone number the user entered and that could cause confusion.

Reply:
```suggestion
        raise InvalidPhoneError("This mobile number is too long")
```